doc/source/microsoft/step-zero-azure.rst

```diff
@@ -121,32 +121,111 @@ If you prefer to use the Azure portal see the `Azure Kubernetes Service quicksta
       This command will also print out something to your terminal screen. You
       don't need to do anything with this text.
 
+#. Create a virtual network and sub-network.
+
+   Kubernetes does not by default come with a controller that enforces ``networkpolicy`` resources.
+   ``networkpolicy`` resources are important as they define how Kubernetes pods can securely communicate with one another and the outside sources, for example, the internet.
+
+   To enable this in Azure, we must first create a `Virtual Network <https://docs.microsoft.com/en-gb/azure/virtual-network/virtual-networks-overview>`_ with Azure's own network policies enabled.
+
+   This section of the documentation is following the Microsoft Azure tutorial on `creating an AKS cluster and enabling network policy <https://docs.microsoft.com/en-us/azure/aks/use-network-policies#create-an-aks-cluster-and-enable-network-policy>`_, which includes information on using `Calico <https://docs.projectcalico.org>`_ network policies.
+
+   .. code-block:: bash
+
+      az network vnet create \
+          --resource-group <RESOURCE-GROUP-NAME> \
+          --name <VNET-NAME> \
+          --address-prefixes 10.0.0.0/8 \
+          --subnet-name <SUBNET-NAME> \
+          --subnet-prefix 10.240.0.0/16
+
+   where:
+
+   * ``--resource-group`` is the ResourceGroup you created
+   * ``-name`` is the name you want to assign to your virtual network, for example, ``hub-vnet``
+   * ``--address-prefixes`` are the IP address prefixes for your virtual network
+   * ``--subnet-name`` is your desired name for your subnet, for example, ``hub-subnet``
+   * ``--subnet-prefixes`` are the IP address prefixes in `CIDR format <https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing>`_ for the subnet
+
+   We will now retrieve the application IDs of the VNet and subnet we just created and save them to bash variables.
+
+   .. code-block:: bash
+
+      VNET_ID=$(az network vnet show
+          --resource-group <RESOURCE-GROUP-NAME>
+          --name <VNET-NAME>
+          --query id
+          --output tsv)
+      SUBNET_ID=$(az network vnet subnet show
+          --resource-group <RESOURCE-GROUP-NAME>
+          --vnet-name <VNET-NAME>
+          --name <SUBNET-NAME>
+          --query id
+          --output tsv)
```

Comment:
Will this work to paste in a terminal without use of `\` etc?

I think it may end up saying `--resource-group` command not found or similar.

Reply:
You're right. I thought the brackets would mitigate this, but it doesn't :)

Reply:
Resolved in https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/1527/commits/fb2d36196afcacc0e11dffc1762ff37edce5c361